packages/apps/transfer/src/components/Global/Grid/GridCol.tsx

```diff
@@ -0,0 +1,34 @@
+import {StyledGridCol} from './styles'
+
+import {IGridColMediaStyles, IGridColMediaType} from "@/types/Grid";
+import {getColumnStyles} from "@/utils/styles";
+import {CSS} from "@stitches/react/types/css-util";
+import React, {FC, ReactNode} from 'react'
+
+export interface IGridRowProps {
+  xs?: IGridColMediaType,
+  sm?: IGridColMediaType,
+  md?: IGridColMediaType,
+  lg?: IGridColMediaType,
+  xl?: IGridColMediaType,
+  xxl?: IGridColMediaType,
+  className?: string
+  children: ReactNode
+}
+
+const GridCol: FC<IGridRowProps> = ({children, className, xs = {}, sm={}, md={}, lg={}, xl={}, xxl = {}}) => {
```

Comment:
can't make it dynamic cause it'll be a runtime type checker so the build will fail